code/processes/gateway.q

```diff
@@ -100,10 +100,16 @@ queryqueue:([queryid:`u#`long$()] time:`timestamp$(); clienth:`g#`int$(); query:
 // client details
 clients:([]time:`timestamp$(); clienth:`g#`int$(); user:`symbol$(); ip:`int$(); host:`symbol$())
 
+//Function to generate random placeholder
+genrand:{system "S";-1?0Ng}
+
+//Generate random placeholder
+placehold:.gw.genrand[]
```

Comment:
Should a different random value be returned each time or does it not really matter what the value is?

Reply:
When I spoke to Jonny last, he said that the GUID could just be generated upon start-up. I assume that this method is what he meant, but I can check with him.

Reply:
That's all good then, I will mark as resolved.